react-ai-chatbot/src/components/Loader/Loader.module.css

```
@@ -16,6 +17,7 @@
     aspect-ratio: 2;
     height: 30px; /* fallback for older browsers */
     --_g: no-repeat radial-gradient(circle closest-side, #0d0d0d 90%, #0d0d0d00);
+    --_g: no-repeat radial-gradient(circle closest-side, light-dark(#0d0d0d, #ececec) 90%, light-dark(#0d0d0d00, #ececec00));
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--_g is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)